src/pages/home/LoginButtons.vue

```diff
@@ -0,0 +1,823 @@
+            accountNameHasError.value = false;
+            accountNameHasWarning.value = false;
+            accountNameIsSuccessful.value = false;
+            accountNameWarningText.value = '';
+            accountNameErrorMessage.value = '';
+            console.log(`----- ${accountNameModel.value}(${currentCount}) --------`);
+
+            if (!settings.isNative()) {
+                return;
+            }
+            const nativeSettings = settings as NativeChainSettings;
+            if (newVal.length === 0) {
+                return;
+            }
+
+            // let's check if the name has only valid characters
+            const validChars = 'abcdefghijklmnopqrstuvwxyz12345';
+            for (let i = 0; i < newVal.length; i++) {
+                const char = newVal[i];
+                if (char === '.') {
+                    accountNameHasError.value = true;
+                    accountNameErrorMessage.value = 'Name cannot contain dots';
+                    console.log(`(${currentCount})`, '- error:', accountNameErrorMessage.value);
+                    return;
+                }
+                if (!validChars.includes(char)) {
+                    accountNameHasError.value = true;
+                    accountNameErrorMessage.value = `invalid character '${char}'`;
+                    console.log(`(${currentCount})`, '- error:', accountNameErrorMessage.value);
+                    return;
+                }
+            }
+
+            // let's check if the name has 12 characters
+            if (newVal.length !== 12) {
+                accountNameHasError.value = true;
+                accountNameErrorMessage.value = `${newVal.length} of 12 characters`;
+                console.log(`(${currentCount})`, '- error:', accountNameErrorMessage.value);
+                return;
+            }
+
+            accountNameIsLoading.value = true;
+            const isAvailable = await nativeSettings.isAccountNameAvailable(newVal);
+            if (currentCount < accountTriesCount.value) {
+                console.log(`(${currentCount}) nos vamos porque el contador es: ${accountTriesCount.value}`);
+                return;
+            }
+
+            accountNameIsLoading.value = false;
+            if (isAvailable) {
+                accountNameHasError.value = false;
+                accountNameHasWarning.value = false;
+                accountNameIsSuccessful.value = true;
+                accountNameWarningText.value = '';
+                accountNameErrorMessage.value = '';
+                console.log(`(${currentCount}) is good!`);
+            } else {
+                accountNameHasError.value = true;
+                accountNameHasWarning.value = false;
+                accountNameIsSuccessful.value = false;
+                accountNameWarningText.value = '';
+                accountNameErrorMessage.value = 'Name is taken';
+                console.log(`(${currentCount})`, '- error:', accountNameErrorMessage.value);
```

Comment:
1. remove console.logs
2. update error messages to use i18n (or at least create a ticket to do so)